R/get_syllabus.R

```diff
@@ -87,7 +87,7 @@ get_timings <- function(ep) {
     }
     return(as.integer(i))
   }
-  times <- c(coerce_integer(yaml$teaching), coerce_integer(yaml$exercises))
+  times <- c(coerce_integer(yaml$teaching), coerce_integer(yaml$exercises), coerce_integer(yaml$break))
```

Comment:
This will set the default break time to zero so that we don't throw any warnings or add extra time to the schedule for lessons that do not have the explicit break yaml item.

```suggestion
  times <- c(coerce_integer(yaml$teaching), coerce_integer(yaml$exercises), coerce_integer(yaml$break, 0L))
```

Reply:
For reference, the default time was added in version 0.11.6: https://github.com/carpentries/sandpaper/pull/396